packages/runtime/src/controls/image.ts

```diff
@@ -2,10 +2,24 @@ export type ImageControlData = string
 
 export const ImageControlType = 'makeswift::controls::image'
 
-type ImageControlConfig = { label?: string }
+export const ImageControlValueFormat = {
+  URL: 'URL',
```

Comment:
Something I was doing to make sure that enums are used and people don't hard-code a value and also to scope things was using strings like `makeswift::controls::image::format::url`, etc.

Reply:
Very cool. Changed!